packages/terra-hookshot/src/terra-dev-site/doc/hookshot/Hookshot.1.doc.mdx

```diff
@@ -0,0 +1,48 @@
+import { Badge } from 'terra-hookshot/package.json?dev-site-package';
+
+import HookshotSrc from 'terra-hookshot/src/Hookshot?dev-site-props-table';
+import HookshotContentSrc from 'terra-hookshot/src/HookshotContent?dev-site-props-table';
+import HookshotExample from '../example/HookshotExample?dev-site-example';
+import BoundedHookshotExample from '../example/BoundedHookshotExample?dev-site-example';
+import CoordsHookshotExample from '../example/CoordsHookshotExample?dev-site-example';
+
+<Badge />
+
+# Terra Hookshot
+
+The Terra Hookshot component positions content according to a targeted attachment, ensuring they stay connected.
+
+Hookshot is currently utilized in the following component(s):<br />
+[Popup][1]
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-hookshot`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+[1]: https://engineering.cerner.com/terra-framework/components/terra-popup/popup/popup
+
+[View Component Source Code](https://github.com/cerner/terra-framework/tree/master/packages/terra-hookshot)
+
```

Comment:
Did we have a usage section for quick reference to the import?

Reply:
Added 72aee83